src/bundle/Resources/public/scss/fieldType/edit/_base-field.scss

```diff
@@ -236,3 +234,13 @@
         }
     }
 }
+
+.ibexa-form-field,
+.ibexa-data-form-field {
+    margin-bottom: calculateRem(24px);
+
+    &:last-child,
+    &--last {
```

Comment:
Why do we need both selectors?

Reply:
Sometimes there are hidden elements at the end so last-child not always work